mvp/README.md

```diff
@@ -0,0 +1,19 @@
+
+**User-Centric** design enables the continuous delivery of value to users. This means that putting a good, functional, tool in the hands of users quickly is more valuable than delivering a perfect tool when it's too late.
+
```

Comment:
This is a _multiline comment_, and it looks like there is also a <ctrl+g> **button: insert a suggestion**